infrastructure/modules/postgresql-flexible/variables.tf

```diff
@@ -81,6 +81,12 @@ variable "sku_name" {
   type        = string
 }
 
+variable "postgres_enable_high_availability" {
```

Comment:
The module is only for postgres. Maybe rename simply to enable_high_availability ?